pkg/lint/lint_test.go

```diff
@@ -62,9 +62,13 @@ func TestBadChart(t *testing.T) {
 			if strings.Contains(msg.Err.Error(), "directory name (badchartfile) and chart name () must be the same") {
 				e3 = true
 			}
+
+			if strings.Contains(msg.Err.Error(), "apiVersion is required") {
```

Comment:
thought: the comment below about typed error values from `.Validate` could be a nice refactor here.

Reply:
I added the guidance as you suggested.